base/testfiles-1run/github-1238.lvt

```diff
@@ -0,0 +1,18 @@
+\input{test2e}
+\START
+\AUTHOR{Jonathan P. Spratte}
+
+\OMIT
+\makeatletter
+\def\@unusedoptionlist{foo,bar,baz}
+\def\@raw@classoptionslist{foo =foo, bar=bar, baz =baz}
+\def\@currname{github-1238}
+\def\@currext{sty}
```

Comment:
I think it's OK to have a `filecontents` env in a test, to avoid all the internal macros used here.

There're already plenty uses of `filecontents` in existing test files.
```
$ rg --no-heading -F '\begin{filecontents}' -g 'testfiles*/**' | wc -l

     182
```

Reply:
True but I think not a blocker